contrib/gitian-descriptors/gitian-win.yml

```diff
@@ -52,7 +52,7 @@ script: |
   for prog in ${FAKETIME_PROGS}; do
     echo '#!/usr/bin/env bash' > ${WRAP_DIR}/${prog}
     echo "REAL=\`which -a ${prog} | grep -v ${WRAP_DIR}/${prog} | head -1\`" >> ${WRAP_DIR}/${prog}
-    echo 'export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/faketime/libfaketime.so.1' >> ${WRAP_DIR}/${prog}
+    echo "export LD_PRELOAD=/usr/lib/$(bitcoin/depends/config.guess)/faketime/libfaketime.so.1" >> ${WRAP_DIR}/${prog}
```

Comment:
Concept ACK

Could maybe assign the native architecture to a variable once at the beginning of the scripst, the use that every time. Invoking this detection every time seems a bit wasteful.

Reply:
BTW, what faketime itself seems to do is
`LD_PRELOAD='/usr/$LIB/faketime/libfaketime.so.1' `
is this a trick, does the dynamic linker interpret a literal $LIB?

(found this out with
```
$ faketime 0 'sh' '-c' env |grep LD_PRELOAD
LD_PRELOAD=/usr/$LIB/faketime/libfaketime.so.1
```
)

Reply:
thanks, done